src/orange/construct/OrangeInputIO.json.cc

```diff
@@ -154,7 +154,8 @@ void from_json(nlohmann::json const& j, VolumeInput& value)
     // Parse bbox
     if (j.contains("bbox"))
     {
-        j.at("bbox").get_to(value.bbox);
+        auto const& bbox = j.at("bbox");
+        value.bbox = {bbox.at(0).get<Real3>(), bbox.at(1).get<Real3>()};
```

Comment:
The original should be OK as long as `orange/BoundingBoxIO.json.hh` is included, right? Or is the format slightly different? We should make sure it's consistent.

Reply:
It looks like the code in `orange/BoundingBoxIO.json.hh` was broken, so I fixed it.